iOS/Core/PixelEvent.swift

```diff
@@ -1258,6 +1258,12 @@ extension Pixel {
         case aiChatMetricSentPromptOngoingChat
         case aiChatInternalSwitchBarDisplayed
 
+        // MARK: Privacy Pro Toolbar Button
+        case privacyProToolbarButtonShown
+        case privacyProToolbarButtonPopoverShown
+        case privacyProToolbarButtonPopoverNoThanksClicked
+        case privacyProToolbarButtonPopoverTryForFreeClicked
+
```

Comment:
These are defined here in an iOS target file but they should be defined in a macOS target file (maybe `PrivacyProPixel.swift`?).